src/output_lib/actions/RespondToFormActions.js

```diff
@@ -1,4 +1,4 @@
-import { Form, FormSubmission } from "../api";
+import { createForm, FormSubmission } from "../api";
```

Comment:
I think `createFormApi` is a better name. And won't we need this for `createFormSubmissionApi` as well eventually?